kuma/core/tests/test_views.py

```diff
@@ -95,31 +95,12 @@ def test_global_home(self):
         doc = pq(r.content)
         assert m.message == doc.find('div.global-notice').text()
 
-    def test_subsection(self):
-        m = Message(message='Search', is_global=False, is_active=True,
-                    url='/search/')
-        m.save()
-
-        url = reverse('search')
-        r = self.client.get(url, follow=True)
-        assert 200 == r.status_code
-
-        doc = pq(r.content)
-        assert m.message == doc.find('div.global-notice').text()
-
-        url = reverse('home')
-        r = self.client.get(url, follow=True)
-        assert 200 == r.status_code
-
-        doc = pq(r.content)
-        assert not doc.find('div.global-notice')
-
```

Comment:
Lemme explain why I deleted this test. 
It relies on going to two different pages. Pages that aren't a wiki document. 
But you can't go to the `/search/` page any more and get a 200 if you don't have a `?q=....` so that page can't be used to test the functionality of Soapbox.
Second, who cares about Soapbox. Pretty sure it works and is never being worked on. 
Third, actually, the error that happens isn't that `request.GET.get('q')` was falsy but that the Elasticsearch query failed hard because the Elasticsearch index doesn't exist in the `elasticsearch` in `docker-compose`. It's a wrong to have any test rely on that database unless it's under the fixtures in the `ElasticTestCase` base class.